app/cars.py

```diff
@@ -0,0 +1,16 @@
+from math import sqrt
+
+
+class Car:
+    fuel_price = 0

```

Comment:
There is no need to store it as a class attribute here. Fuel price depends on where are you going. You don't need to store this value outside or a function or method where you use it

Reply:
You didn't fix that